### ISSUE TITLE
Space between right parenthesis and opening curly bracket enforces in regex patterns

### ISSUE THREAD

**Issue by TCABJ** [jimhester/lintr#452]
Regex patterns can have a right parenthesis and an opening curly bracket next to each other, but this is picked up as a styling issue by the linter.

Test code
```R
regex_pat <- "(12|13){0,2}"
```

Resulting message:
```
test.R: style: There should be a space between right parenthesis and an opening curly brace.
"(12|13){0,2}"
```

**Comment by russHyde** [comment]
Thanks for reporting, but this is a duplicate of https://github.com/jimhester/lintr/issues/441 . Hence, closing
